Core/GUI/GameTooltipHooks.lua

```diff
@@ -515,14 +515,20 @@ local function onTooltipSetItem(tooltip, tooltipData)
 		return
 	end
 
-	local _, itemLink = tooltip:GetItem()
-	if type(itemLink) ~= "string" then
+	local name, link = tooltip:GetItem()
+	if type(link) ~= "string" or not name then
```

Comment:
Just to make sure I understand correctly: The failure case here is that `GetItem()` returns `nil, link` where `link` is an actual item link? That seems rather peculiar...